examples/buildconfig/docker-compose.yml

```diff
@@ -3,5 +3,6 @@ version: "2"
 services:
     foo:
         build: "./build"
+        image: "localhost:5000/vagrant/foo:latest"
         command: "sleep 3600"
```

Comment:
are there plans to expose options like "no-cache" and "force-pull" for performing the docker build?